### PR TITLE
javascript2-week2/Sevda

### PR DESCRIPTION
**Pull request by Sevda-NZ**
Due to severe sickness couldn't finish the last 2 parts of HYFBAY. Will try to finish it tomorrow.
Thanks for the review 🙏

### CODE REVIEW

**Comment by szoboszlaianna** [comment]
Could you move the movies array to its own file "movies.js" and then import that in the other js file? This way it is really hard to review it because github won't open a file with this many lines. Thanks! :)

File: 3_javascript/javascript2/week2/Week2-HYF/main.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  );
+  renderProducts(filteredProducts);
+});
+
+function renderProducts(products) {
+  productsUl.innerHTML = "";
+  products.forEach((product) => {
+    const li = document.createElement("li");
+    let shipsToHTML = "";
+    product.shipsTo.forEach(
```
Comment (szoboszlaianna) [inline 3_javascript/javascript2/week2/Week2-HYF/main.js:51]: this fails because products don't have a shipsto list.

**Review by szoboszlaianna — CHANGES_REQUESTED** [review]

File: 3_javascript/javascript2/week2/Week2-HYF/main.js (hunk 1 above)
Comment (Sevda-NZ) [reply 3_javascript/javascript2/week2/Week2-HYF/main.js:51]: I am still working on it!

**Comment by Sevda-NZ** [comment]
Made it functional   ✌🏻

**Pull request closed without merging**